### DISCUSSION TITLE
Designing Mechanics: Fungi

### DISCUSSION THREAD

**Discussion by wallamide** [Leafwing-Studios/Emergence#145] [Design notes]
# Making an "Organic" Factory Sim

As we lay the framework for basic game functionality, we need to consider ways to use biological systems to inspire our game mechanics. Here is a discussion to start planning some mechanics for fungi and species we may wish to mimic/use. 

## Current plans

Currently we plan for fungi to produce food for ants by modifying plant biomass. This degradation mechanic is planned to be expanded on by allowing fungi to produce compost. 

## Potential Starting Points

Fungi are well known as decomposers but also do many other things in their environments:

- nutrient fixation and transport to plants, 
- nematode removal
- soil "cleaning" (in partnership with bacteria),  and
- food (as fruiting bodies or edidile secretions) 
- parasite that controls host metabolism/behavior 

There are also many ways that fungi traverse the world, reproduce, and maintain physical forms. This gives us a lot of potential ways to incorporate different types of fungi to produce/modify various resources.

**Comment by alice-i-cecile** [comment]
# Mycorrhizae

Extracting nutrients from the soil, and sharing nutrients and water in exchange for energy. This is a very interesting life strategy for fungi, and very useful as a power-pole substitute!

**Comment by wallamide** [comment]
# Parasitic Fungi

Due to the (formerly) massive amount of insects, there are many fungi that have evolved to form parasitic relationships with many types of invertebrates and vertebrates. Below is a list, in no special order, of some of fungi that have specialized to parasitize: 

## *Ophiocordyceps unilateralis*

The "zombie-ant fungus", known to parasitize ants of the [tribe](https://en.wikipedia.org/wiki/Tribe_(biology)) [Camponotini](https://en.wikipedia.org/wiki/Camponotini). *O. unilateralis* spores get on an ant and grow hyphae that piece through the ant's cuticle and make their way throughout the ant. Once besides the brain of the ant, *O. unilateralis* begins to modulate ant behavior, causing the ant to fall from trees and instead make their way up a smaller plant on the forest floor where they rest on the underside of the leaf. Fungal mycelium then infiltrates the jaw of the ant where it causes the ant to present lock law like symptoms before dying. A spore producing fungal body then pops out of the ant's head and spreads spores below. 

## *Harposporium anguillulae*

A nematode parasite, *H. anguilulae* produces crescent shaped spores (conidia) that get lodged in the upper GI tract of nematodes. Spores then use this warm, damp environment to grow mycelial networks that break through the nematode cuticle before forming spore-producing biochemical machinery and the cycle starts again. 

## *Entomophthora muscae*

A fly parasite, *E muscae* produces conidia that germinate on flies, produce a germ tube that then burrows into the fly's cuticle. The [germ tube](https://en.wikipedia.org/wiki/Germ_tube) then reaches the [haemolymph](https://en.wikipedia.org/wiki/Hemolymph) and mycelium begins to grow throughout the fly until reaching the brain, causing the fly to then land and crawl upwards. The fly comes to a resting spot where the fungus consumes the fly before producing spore machinery and the cycle starts again. 

## Potential Applications/Considerations

Adding parasitic fungi allows for a mechanic where both friendly and hostile animal units will have the potential to get infected. This means the player has to ensure ways for their colony to "get clean" to prevent fungal germination and infection, while also offering a chance the player may be aided by a non-ant specific fungus that parasitizes enemy units. 

- Parasite spread can be random and rare, with various factors that contribute to infection risk.
- Some species of parasitic fungi have their own parasites that we could offer as a sp. of fungus the hive mind cultivates to prevent infection
- Some parasitic fungi mentioned have relatives able to parasitize non-parasitic fungi as well. 
- We can fine tune lots of aspects of parasite mechanics.

**Comment by wallamide** [comment]
# Fungal Materials

Mycelium is a very dense and anti-microbial surface. Certain species of [basidiocarp]https://en.wikipedia.org/wiki/Basidiocarp() fungi have been shown to be usable as [paper-like materials, insultaion, textiles, and leather substitues](https://www.sciencedirect.com/science/article/abs/pii/S0167779921000603), with [this table](https://fungalbiolbiotech.biomedcentral.com/articles/10.1186/s40694-022-00134-x/tables/1) from a [fungal matearial risk assesment](https://fungalbiolbiotech.biomedcentral.com/articles/10.1186/s40694-022-00134-x) shows the wide variety of species and application types. 

## Using Fungal Materials

Of note, many species are able to be used to create concrete/wood/other construction materials that allow for the creation of various structures. This would allow us to create a mechanic where, once the hive mind's colony has found certain species of fungi, they are able to cultivate their waste with the fungi to make building materials that let the player recycle plants/other fungi/waste into new structures or intermediate resource. 

Fungal materials could also give us one mechanic to manage the potential ways random damage to a supply chain/process can be managed. Because fungi need to be cultivated, this also allows for a timing component where the player needs to weigh their choice given the way some other options for fixing the damage will be faster than fungi but may create more waste or make other aspects of maintaining the factory/automation/supply chain more difficult.
